src/Sylius/Bundle/CoreBundle/Resources/config/app/sylius.yml

```diff
@@ -181,6 +181,7 @@ sylius_inventory:
         inventory_unit:
             classes:
                 model: %sylius.model.order_item_unit.class%
+                repository: Sylius\Bundle\CoreBundle\Doctrine\ORM\OrderItemUnitRepository
```

Comment:
Why did you added it here? I guess it should be part of default configuration, so it should be placed under `Resources/config/driver/doctrine/orm.xml`